controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - shoulder_joint
        - elbow_joint
        - gripper_joint
      command_interfaces:
        - position
      state_interfaces:
        - position
        - velocity